setup.cfg

```diff
@@ -35,9 +35,42 @@ install_requires =
 
 [options.extras_require]
 HE =
-    tenseal==0.3.0
+    tenseal==0.3.12
 PSI =
     openmined.psi==1.1.1
+PT =
+    torch
+    torchvision
+    tensorboard
+SKLEARN =
+    sklearn
+TRACKING =
+    mlflow
+app_opt =
+    %(HE)s
```

Comment:
the setup.cfg follows: https://docs.python.org/3/library/configparser.html#configparser.BasicInterpolation

So you can treat this as some "replacement", it will substitute this with requirements specified under "HE"